src/test/e2e-emulated/contracts/mutating-methods.tact

```diff
@@ -64,4 +66,9 @@ contract Tester {
         self.s.loadUint(1);
         return self.s.loadUint(3);
     }
+
+    get fun test10(dict: Cell?): Cell? {
+        dict.nativeUdictStoreUint(8, 123, rawSlice("456"));
```

Comment:
Let's check `nativeUdictStoreUint` can be chained with itself

Reply:
how and why should it be chained in this case? the function itself returns `void` in this scenario.

Reply:
ah, yes, my bad